not-sure-if-bug: strange mypy behavior in the latest sqlglot

```python
from sqlglot import parse_one, exp

a = parse_one("SELECT * FROM t").find(exp.Table)
print(a)
```

```bash
$ python -m mypy test.py
test.py:3: error: <nothing> has no attribute "find"  [attr-defined]
Found 1 error in 1 file (checked 1 source file)
```

Could be a mypy bug or the usual mypy idiocy, but just FYI.

--- a/sqlglot/__init__.py
+++ b/sqlglot/__init__.py
@@ -84,58 +84,17 @@ def parse(sql: str, read: DialectType = None, **opts) -> t.List[t.Optional[Expre
 
 
 @t.overload
-def parse_one(
-    sql: str,
-    read: None = None,
-    into: t.Type[E] = ...,
-    **opts,
-) -> E:
+def parse_one(sql: str, *, into: t.Type[E], **opts) -> E:
     ...
 
 
 @t.overload
-def parse_one(
-    sql: str,
-    read: DialectType,
-    into: t.Type[E],
-    **opts,
-) -> E:
-    ...
-
-
-@t.overload
-def parse_one(
-    sql: str,
-    read: None = None,
-    into: t.Union[str, t.Collection[t.Union[str, t.Type[Expression]]]] = ...,
-    **opts,
-) -> Expression:
-    ...
-
-
-@t.overload
-def parse_one(
-    sql: str,
-    read: DialectType,
-    into: t.Union[str, t.Collection[t.Union[str, t.Type[Expression]]]],
-    **opts,
-) -> Expression:
-    ...
-
-
-@t.overload
-def parse_one(
-    sql: str,
-    **opts,
-) -> Expression:
+def parse_one(sql: str, **opts) -> Expression:
     ...
 
 
 def parse_one(
-    sql: str,
-    read: DialectType = None,
-    into: t.Optional[exp.IntoType] = None,
-    **opts,
+    sql: str, read: DialectType = None, into: t.Optional[exp.IntoType] = None, **opts
 ) -> Expression:
     """
     Parses the given SQL string and returns a syntax tree for the first parsed SQL statement.

